sdk/digitaltwins/azure-digitaltwins-core/API design.md

```diff
@@ -1254,18 +1251,10 @@ Sync APIs
  /**
      * Creates one or many models.
      * @param models The list of models to create. Each string corresponds to exactly one model.
-     * @return The list of created models
-     */
-    @ServiceMethod(returns = ReturnType.COLLECTION)
-    public List<ModelData> createModels(List<String> models) { }
-
-    /**
-     * Creates one or many models.
-     * @param models The list of models to create. Each string corresponds to exactly one model.
-     * @return A REST response containing the list of created models.
+     * @return A {@link PagedIterable} of created ModelData.
```

Comment:
Add a comment about context too?

Reply:
I will only update the actual implementation from now on

Reply:
> What is part 2 suppose to contain?
> Decommision and rest of it?

Decommission APIs and the Samples and tests.